the-journals-of-gerontology-series-a.csl

```diff
@@ -0,0 +1,260 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US" page-range-format="expanded">
+  <info>
+    <title>The Journals of Gerontology: Series A</title>
+    <title-short/>
+    <id>http://www.zotero.org/styles/the-journals-of-gerontology-series-a</id>
+    <link href="http://www.zotero.org/styles/the-journals-of-gerontology-series-a" rel="self"/>
+    <link rel="template" href="http://www.zotero.org/styles/american-medical-association"/>
+    <link href="http://www.oxfordjournals.org/our_journals/gerona/for_authors/general.html" rel="documentation"/>
+    <link href="http://westlibrary.txwes.edu/sites/default/files/pdf/ama_citation_style.pdf" rel="documentation"/>
```

Comment:
Please delete the line above with the http://westlibrary.txwes.edu/sites/default/files/pdf/ama_citation_style.pdf link (which also happens to be broken).